### ISSUE TITLE
kola: rawhide: 20220209: selinux denials causing `ext.config.chrony.dhcp-propagation` failure

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#1097]
The `ext.config.chrony.dhcp-propagation` test is failing. It appears the new `selinux-policy-36.1-1.fc36` is bringing in some new policy causing the failures to occur. Here is the test failure from kola:

```
=== RUN   ext.config.chrony.dhcp-propagation                                                                                                                                                                             
systemctl status kola-runext.service:                                                                                                                                                                                    
● kola-runext.service                                                                                                                                                                                                    
     Loaded: loaded (/etc/systemd/system/kola-runext.service; static)                                                                                                                                                    
     Active: deactivating (stop-sigterm) (Result: exit-code) since Wed 2022-02-09 16:52:44 UTC; 1s ago                                                                                                                   
    Process: 1536 ExecStart=/usr/local/bin/kola-runext-dhcp-propagation (code=exited, status=1/FAILURE)                                                                                                                  
   Main PID: 1536 (code=exited, status=1/FAILURE)                                                                                                                                                                        
      Tasks: 2 (limit: 1050)                                                                                                                                                                                             
     Memory: 135.5M                                                                                                                                                                                                      
        CPU: 8.022s                                                                                                                                                                                                      
     CGroup: /system.slice/kola-runext.service                                                                                                                                                                           
             └─2440 /usr/bin/conmon --api-version 1 -c 4020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac819910fc9 -u 4020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac819910fc9 -r /usr/bin/crun -b /var/lib/
containers/storage/overlay-containers/4020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac819910fc9/userdata -p /run/containers/storage/overlay-containers/4020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac8199
10fc9/userdata/pidfile -n dnsmasq --exit-dir /run/libpod/exits --full-attach -s -l journald --log-level warning --runtime-arg --log-format=json --runtime-arg --log --runtime-arg=/run/containers/storage/overlay-contain
ers/4020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac819910fc9/userdata/oci-log --conmon-pidfile /run/containers/storage/overlay-containers/4020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac819910fc9/userda
ta/conmon.pid --exit-command /usr/bin/podman --exit-command-arg --root --exit-command-arg /var/lib/containers/storage --exit-command-arg --runroot --exit-command-arg /run/containers/storage --exit-command-arg --log-le
vel --exit-command-arg warning --exit-command-arg --cgroup-manager --exit-command-arg systemd --exit-command-arg --tmpdir --exit-command-arg /run/libpod --exit-command-arg --cni-config-dir --exit-command-arg /etc/cni/
net.d/ --exit-command-arg --network-backend --exit-command-arg cni --exit-command-arg --runtime --exit-command-arg crun --exit-command-arg --storage-driver --exit-command-arg overlay --exit-command-arg --storage-opt -
-exit-command-arg overlay.mountopt=nodev,metacopy=on --exit-command-arg --events-backend --exit-command-arg journald --exit-command-arg container --exit-command-arg cleanup --exit-command-arg --rm --exit-command-arg 4
020263bcf0a4adfc4de8acbff0fbb0af89e3f5a8bb0412f10546ac819910fc9                                                                                                                                                          
                                                                                                                                                                                                                         
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: + [[ MS Name/IP address         Stratum Poll Reach LastRx Last sample                                                                                          
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: ===============================================================================                                                                                
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: ^+ clock.fmt.he.net              1   6   377    19  -1269us[-1269us] +/-   37ms                                                                                
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: ^* lithium.constant.com          2   6   377    21   +123us[ +136us] +/-   38ms                                                                                
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: ^+ li1210-167.members.linod>     2   6   377    19   -776us[ -776us] +/-   65ms                                                                                
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: ^+ nu.binary.net                 2   6   377    19  +1740us[+1740us] +/-   52ms != *\t\i\m\e\-\c\-\g\.\n\i\s\t\.\g\o\v* ]]                                     
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: + echo 'waiting for ntp server to appear'         
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: waiting for ntp server to appear                  
Feb 09 16:52:42 qemu0 kola-runext-dhcp-propagation[1536]: + sleep 1                                                                                                                                                      
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + retries=1                                                                                                                                                    
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + [[ 1 -gt 0 ]]                                                                                                                                                
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + [[ ! MS Name/IP address         Stratum Poll Reach LastRx Last sample        
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ===============================================================================
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ clock.fmt.he.net              1   6   377    19  -1269us[-1269us] +/-   37ms                                           
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^* lithium.constant.com          2   6   377    21   +123us[ +136us] +/-   38ms                                                                                
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ li1210-167.members.linod>     2   6   377    19   -776us[ -776us] +/-   65ms
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ nu.binary.net                 2   6   377    19  +1740us[+1740us] +/-   52ms =~ time-c-g\.nist\.gov ]]
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[3113]: ++ chronyc sources
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + NTPSERVER='MS Name/IP address         Stratum Poll Reach LastRx Last sample
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ===============================================================================
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ clock.fmt.he.net              1   6   377    20  -1269us[-1269us] +/-   37ms
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^* lithium.constant.com          2   6   377    22   +123us[ +136us] +/-   38ms
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ li1210-167.members.linod>     2   6   377    20   -776us[ -776us] +/-   65ms
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ nu.binary.net                 2   6   377    20  +1740us[+1740us] +/-   52ms'                                                                               
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + [[ MS Name/IP address         Stratum Poll Reach LastRx Last sample                                    
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ===============================================================================
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ clock.fmt.he.net              1   6   377    20  -1269us[-1269us] +/-   37ms
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^* lithium.constant.com          2   6   377    22   +123us[ +136us] +/-   38ms
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ li1210-167.members.linod>     2   6   377    20   -776us[ -776us] +/-   65ms                                                                                
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: ^+ nu.binary.net                 2   6   377    20  +1740us[+1740us] +/-   52ms != *\t\i\m\e\-\c\-\g\.\n\i\s\t\.\g\o\v* ]]                                     
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + echo 'waiting for ntp server to appear'                                                                                                                      
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: waiting for ntp server to appear                                                                                                                               
Feb 09 16:52:43 qemu0 kola-runext-dhcp-propagation[1536]: + sleep 1                                                                                                                                                      
Feb 09 16:52:44 qemu0 kola-runext-dhcp-propagation[1536]: + retries=0                                                                                                                                                    
Feb 09 16:52:44 qemu0 kola-runext-dhcp-propagation[1536]: + [[ 0 -gt 0 ]]                                                                                                                                                
Feb 09 16:52:44 qemu0 kola-runext-dhcp-propagation[1536]: + '[' 0 -eq 0 ']'                                                                                                                                              
Feb 09 16:52:44 qemu0 kola-runext-dhcp-propagation[1536]: + echo 'propagation of ntp server information via dhcp failed'                 
Feb 09 16:52:44 qemu0 kola-runext-dhcp-propagation[1536]: propagation of ntp server information via dhcp failed                                                                     
Feb 09 16:52:44 qemu0 kola-runext-dhcp-propagation[1536]: + exit 1                                 
Feb 09 16:52:44 qemu0 systemd[1]: kola-runext.service: Main process exited, code=exited, status=1/FAILURE
--- FAIL: ext.config.chrony.dhcp-propagation (395.80s)                                                      

```

Here are a representative subset of the denials we see:

```
Feb  9 19:04:10.250000 audit[1184]: AVC avc:  denied  { sys_nice } for  pid=1184 comm="nm-dispatcher" capability=23  scontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tcontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tclass=capability permissive=0
Feb  9 19:04:10.250000 audit[1184]: AVC avc:  denied  { setsched } for  pid=1184 comm="nm-dispatcher" scontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tcontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tclass=process permissive=0
Feb  9 19:04:10.256000 audit[1171]: USER_AVC pid=1171 uid=81 auid=4294967295 ses=4294967295 subj=system_u:system_r:system_dbusd_t:s0-s0:c0.c1023 msg='avc:  denied  { send_msg } for  scontext=system_u:system_r:NetworkManager_t:s0 tcontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tclass=dbus permissive=0
```

**Comment by dustymabe** [comment]
This seems to be fixed in [`selinux-policy-36.3-1.fc37`](https://bodhi.fedoraproject.org/updates/FEDORA-2022-4e24511b06)

**Comment by dustymabe** [comment]
Drop over in https://github.com/coreos/fedora-coreos-config/pull/1543
